content/docs/features/bill-of-materials.md

```diff
@@ -4,24 +4,79 @@ summary="A Software `Bill-of-Materials` (`BOM`) gives you a layer-by-layer view
 +++
 
 ## Summary
-A Software `Bill-of-Materials` (`BOM`) gives you a layer-by-layer view of what's inside your container in a variety of formats including `JSON`. Buildpacks can also populate your app image with metadata from the build process, allowing you to audit the app image for information like, the process types that are available and the commands associated with them, buildpacks which were used to create the app image etc. Apart from the above standard metadata, buildpacks can also populate information about the dependencies they have provided in form of a `Bill-of-Materials` (`BOM`).
+
+A Software **Bill-of-Materials** (`BOM`) provides information necessary to know what's inside your container and how it was constructed.
+Cloud Native Buildpacks provide two forms of Bill-of-Materials.
+
+1. Buildpacks can populate `BOM` information about the dependencies they have provided.
+2. A list of what buildpacks were used to build the application.
 
 ## Adding Bill of Materials
+
 Use the following tutorial to add a `Bill-of-Materials` using buildpacks. <br/>
 [Adding bill of materials][adding-bill-of-materials]
 
 ## Viewing Bill of Materials
+
 You can use this command to inspect your app for it's `Bill-of-Materials`.
 
 ```bash
 pack inspect-image your-image-name --bom
 ```
+
 It can also be accessed by looking at the label `io.buildpacks.build.metadata`. For example, running Docker CLI, jq and using the following command.
 
 ```bash
 docker inspect your-image-name | jq -r '.[0].Config.Labels["io.buildpacks.build.metadata"] | fromjson'
 ```
 
+Following is the the information listed in `io.buildpacks.build.metadata` for [Sample Java App](https://github.com/buildpacks/samples/tree/main/apps/java-maven) obtained by building the app using buildpacks and running the above command.
```

Comment:
Do we want to advertise this label to users given that we will soon be moving away from it?

Reply:
Probably just mention the command and remove the description? also why are we moving away from it, any references to learn more about it. :)

Reply:
> Do we want to advertise this label to users given that we will soon be moving away from it?

I always have a conflict with this. Things always change. I'm not a fan of _waiting_ for an upcoming change to block describing the world for what it is now. It seems like a quick follow-up item to remove/replace this once that change comes to fruition. Thoughts?